echo/server/dembrane/report_utils.py

```
@@ -57,8 +57,18 @@ async def get_report_content_for_project(project_id: str, language: str) -> str:
         if conversation["summary"] is None:
-            logger.info(f"Conversation {conversation['id']} has no summary")
-            continue
+            logger.info(f"Conversation {conversation['id']} has no summary, attempting to generate one")
+            
+            result = summarize_conversation(conversation["id"], DirectusSession(user_id="none", is_admin=True))
+            
+            if result["status"] == "success" and result.get("summary"):
+                # Update the conversation object with the new summary
+                conversation["summary"] = result["summary"]
+                
+                logger.info(f"Summary generation result for conversation {conversation['id']}: {result}")
```

Pylint (3.3.7)

[convention] Line too long (105/100)

(C0301)